src/compiler/types.ts

```diff
@@ -2491,6 +2491,7 @@ namespace ts {
     export interface MetaProperty extends PrimaryExpression {
         readonly kind: SyntaxKind.MetaProperty;
         readonly keywordToken: SyntaxKind.NewKeyword | SyntaxKind.ImportKeyword;
+        readonly expression: Token<SyntaxKind.NewKeyword | SyntaxKind.ImportKeyword>;

```

Comment:
`expression` was used to match property access expression.

Reply:
I think I’m missing something important, but I don’t understand why this (and the parser change) was needed. You can get a reference to the token in the services with `node.getChildren()` and friends even if the token isn’t saved into the parse tree.

Reply:
Great point! I was not thinking and completely forgot about that 😅

Reverted.